package.json

```diff
@@ -50,6 +51,76 @@
     "chai": "^3.5.0",
     "vscode-textmate": "^2.1.1"
   },
+  "runtimeDependencies": [
```

Comment:
I've added an `installPath` property to make item 2 easier. Essentially, this is combined with the extension root path to compute the install location.